modules/odata/odata-core/pom.xml

```diff
@@ -21,12 +21,20 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>${maven.compiler.plugin.version}</version>
 				<configuration>
-					<source>${maven.compiler.source}</source>
-					<target>${maven.compiler.target}</target>
+					<source>1.8</source>
```

Comment:
Do we need to set it explicitly to 1.8? Currently it is 11 which supports the features used below

Reply:
The thing is that when a module is compiled with java 11 it cannot be used with projects that run on java 1.8